### PR TITLE
Actually Additions compat

### PR DESCRIPTION
**Pull request by WaitingIdly**
what this PR adds:
- Add Groovyscript compat for the Crusher, Oil Gen, Compost, Ball Of Fur, Treasure Chest, Empowerer, Atomic Reconstructor, and Mining Lens (both Stone and Netherrack types)
- an examples file for the above

### CODE REVIEW

File: src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+        return new SimpleObjectStream<>(ActuallyAdditionsAPI.EMPOWERER_RECIPES)
+                .setRemover(this::remove);
+    }
+
+    public static class RecipeBuilder extends AbstractRecipeBuilder<EmpowererRecipe> {
+
+        private IIngredient mainInput;
+        private int energyPerStand;
+        private int time;
+        private float[] particleColor = new float[]{0, 0, 0};
```
Comment (brachy84) [inline src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java:88]: Is this really how AA stores their color? :concern:

**Review by brachy84 — APPROVED** [review]

File: src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java (hunk 1 above)
Comment (WaitingIdly) [reply src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java:88]: it is, yup. should i add builder methods to edit it in a more conventional manner (eg Color, hexadecimal, r/g/b)?

Comment (Testure) [reply src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java:88]: I would store the color as an `int` value and construct the cursed format when you register the recipe.

Comment (brachy84) [reply src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/Empowerer.java:88]: Yes, please add sane mathod params

**Pull request merged**